sklearn/impute.py

```diff
@@ -935,15 +950,13 @@ def transform(self, X):
                 fit_mode=False
             )
             if not (it + 1) % imputations_per_round:
-                if i_rnd >= self.n_burn_in:
-                    Xt += X_filled
                 if self.verbose > 1:
-                    print('[ChainedImputer] Ending imputation round '
+                    print('[IterativeImputer] Ending imputation round '
                           '%d/%d, elapsed time %0.2f'
-                          % (i_rnd + 1, n_rounds, time() - start_t))
+                          % (i_rnd + 1, self.n_iter, time() - start_t))
                 i_rnd += 1
 
-        Xt /= self.n_imputations
+        Xt = X_filled
```

Comment:
this could be simplified, isn't it?

Reply:
I'll replace `X_filled` with `Xt` everywhere.